.gitignore

```diff
@@ -7,3 +7,6 @@
 # VS Code Files
 **/.vscode
 /.vs
+/src/env
+/src/.vs
```

Comment:
I would update `/.vs` path to `**/.vs` to grab all sub directories and you can remove `/src/.vs`